cms/templates/js/list.underscore

```diff
@@ -1,9 +1,12 @@
 <% if (length === 0) { %>
-    <div class="no-group-configurations-content">
-        <p><%- interpolate(gettext("You have not created any %(item_type)ss yet."), {item_type: itemCategoryDisplayName}, true) %><a href="#" class="button new-button"><i class="icon fa fa-plus"></i> <%= gettext("Add your first content group") %></a></p>
+    <div class="no-content">
+        <p>
+            <%- interpolate(gettext("You have not created any %(item_type)ss yet."), {item_type: itemCategoryDisplayName}, true) %>
```

Comment:
Whoops. @catong: we need an alternative phrasing here. How about "You have not created any %(item_type)s items yet." or "You have not created any %(item_type)s instances yet."?